src/screens/GovernanceProposalDetails/components/ProposalDetails/index.tsx

```diff
@@ -79,6 +65,15 @@ const ProposalDetails: React.FC<ProposalDetailsProps> = ({ proposal }) => {
   return (
     <View>
       <Spacer paddingTop={24} />
+      {/* Summary */}
```

Comment:
@RiccardoM I display the proposal summary above the description. Let me know if this is fine.

Reply:
Yea it's perfect :+1: